src/main/java/com/onarandombox/MultiversePortals/utils/DisplayManager.java

```diff
@@ -0,0 +1,64 @@
+package com.onarandombox.MultiversePortals.utils;
+
+import com.onarandombox.MultiverseCore.api.MultiverseWorld;
+import com.onarandombox.MultiversePortals.MVPortal;
+import com.onarandombox.MultiversePortals.MultiversePortals;
+import org.bukkit.ChatColor;
+import org.bukkit.command.CommandSender;
+
+public class DisplayManager {
```

Comment:
I think `DisplayUtils` will be a better name for this class, I don't think it really manages anything.